### ISSUE TITLE
Bug: Lyrics font should follow size selected by user

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#123]
Cmd-8/9 are used to change font sizes.  Right now, the Lyrics pane does not follow the changes, but it should (everything else does).

**Comment by mpogue2** [comment]
OK, if there's a "cuesheet2.css" file in the directory, it **does** get applied correctly, but then the size is fixed (it will be the one that you specified).  If there's no "cuesheet2.css" file, then the size of the text DOES change along with the Cmd-8/9/0 controls.

**Comment by mpogue2** [comment]
OK, the problem was two-fold:
1) we needed to use ZoomIn and ZoomOut, which are only available on QTextEdit.  We don't need the "follow navigation" feature of QTextBrowser.
2) if the CSS file specifies a numeric size (e.g. 24pt), then that overrides any setting of fontSize with ZoomIn/Out.

So, I changed the cue sheet pane from QTextBrowser to QTextEdit, did ZoomIn/Out along with everything else.  I also changed the behavior of finding cuesheet2.css to always look in the Lyrics directory.  Before it was cue sheet-relative.  I think it's more likely that people will want a global one, that multiple local ones.  I also changed the sample cue sheet to use size names, rather than specific point sizes.
